### ISSUE TITLE
Playground UX improvements

### ISSUE THREAD

**Issue by BawdyInkSlinger** [gillchristian/tsplay.dev#121]
First of all, I want to say this is a great Playground plug-in. Thank you for creating it.

**TL;DR:**

- [ ] Within the list of previous short links, indicate which is currently active/newest.
  - [ ] Maybe a creation date could be displayed within these gray boxes?
  - [ ] Maybe a gray box could be colored differently to represent the current/latest short link? 
- [ ] Modify the "⚠️ Already created a short link for the current code ⚠️" message to explain why that may be the case.

Lately, whenever I click on the "Link Shortener" tab and click on the CREATE SHORT LINK button, it usually reports "⚠️ Already created a short link for the current code ⚠️". I'm not sure why that is happening, but, the plug-in provides a historical list of short links below:

![image](https://github.com/gillchristian/tsplay.dev/assets/51732963/18a36ad2-5f7a-4867-b705-483634ab61ca)

The thing is, I'm not sure which one is the one that was already created. Is it the first? Is it the last? Is it sorted some other way? 

Through experimentation, it seems to be the last one. But I think it would be nice if I didn't have to experiment to learn this; maybe a creation date could be displayed within these gray boxes? Or, maybe one could be colored differently to represent the current/latest playground? 

Thank you!

**Comment by gillchristian** [comment]
@BawdyInkSlinger thanks for the feedback! You are correct that this is quite confusing. We are in the process of re-doing the whole plugin so don't think it makes sense to fix the current version.

I'll make sure to incorporate your feedback in the new version.
